src/stats/hist.jl

```diff
@@ -72,12 +82,19 @@ function Makie.plot!(plot::Hist)
         end
     end
 
-    points = lift(edges, plot.normalization) do edges, normalization
+    points = lift(edges, plot.normalization, plot.offset, plot.scale_to, plot.flip) do edges, normalization, offset, scale_to, flip
         h = StatsBase.fit(StatsBase.Histogram, values[], edges)
         h_norm = StatsBase.normalize(h, mode = normalization)
         centers = edges[1:end-1] .+ (diff(edges) ./ 2)
         weights = h_norm.weights
-        return Point2f0.(centers, weights)
+        if !isnothing(scale_to)
+            min, max = extrema(weights)
+            weights .= ((weights .- min) ./ (max - min)) .* scale_to
```

Comment:
This seems dangerous to me. Have you tried setting `scale` on `hist(rand(1_000_000))` with the current implementation?

I think `weigths .= weights ./ max .* scale_to` is more reasonable. Morally, you always have weight `0` outside your data range anyways, so `min` "should" always be `0`. 

In general, I think the bar length should be proportional to the number of entries in that bin, which you respect with `weigths .= weights ./ max .* scale_to`, but not with the current implementation.

Reply:
Oh good catch! That's absolutely wrong :D